contracts/walletDeployer.sol

```diff
@@ -19,51 +19,28 @@
 pragma solidity ^0.5.10;
 
 import "./wallet.sol";
+import "./walletCache.sol";
 import "./internals/controllable.sol";
-import "./internals/ensResolvable.sol";
 
 //// @title Wallet deployer with pre-caching if wallets functionality.
 contract WalletDeployer is ENSResolvable, Controllable {
 
-    event CachedWallet(Wallet _wallet);
     event DeployedWallet(Wallet _wallet, address _owner);
     event MigratedWallet(Wallet _wallet, Wallet _oldWallet, address _owner);
 
-    /*****   Constants   *****/
     bytes32 public constant controllerNode = 0x7f2ce995617d2816b426c5c8698c5ec2952f7a34bb10f38326f74933d5893697;
-    bytes32 public constant licenceNode = 0xd0ff8bd67f6e25e4e4b010df582a36a0ee9b78e49afe6cc1cff5dd5a83040330;
-    bytes32 public constant tokenWhitelistNode = 0xe84f90570f13fe09f288f2411ff9cf50da611ed0c7db7f73d48053ffc974d396;
+    bytes32 public constant walletCacheNode = 0x7eee9c3927d17f70ce19de05f73d05dbda3449d450ba9a4c64f24c24bfb9d7ac;
```

Comment:
should be passed in constructor

Reply:
same as controller node ...

Reply:
This is now passed in via the constructor, with default values being the ones we want on mainnet.